app/models/report_composer.rb

```diff
@@ -196,6 +214,7 @@ def mail_to
 
   def report_filename
     name = @template.suggested_report_name.to_s
```

Comment:
In case we are using `Mime::Type`:
```suggestion
    name = "#{@template.suggested_report_name}.#{format.to_sym}"
```

Reply:
not sure what you exactly meant here, we're using mime::type now to detect the extension, but that still needs to be added

also not sure, why we'd convert it to_sym and then let it be interpolated back to string